faiss_vector_posting.go

```diff
@@ -403,3 +403,25 @@ func (sb *SegmentBase) InterpretVectorIndex(field string) (segment.VectorIndex,
 	vecIndex, err = faiss.ReadIndexFromBuffer(indexBytes, faiss.IOFlagReadOnly)
 	return wrapVecIndex, err
 }
+
+func (sb *SegmentBase) UpdateFieldStats(stats map[string]map[string]int) {
```

Comment:
i think you can unit test this out

Reply:
Added unit tests in bleve which cover this code path as well as merge